zulipterminal/core.py

```diff
@@ -3,6 +3,7 @@
 import os
```

Comment:
You probably want to place the #241 reference in some text below the commit summary, as something like "Fixes #241", which also has the benefit that it will close that issue when this commit is merged :)

Reply:
@realsdx This is optional, but I noticed you didn't have this, so just a reminder in case you'd like to.